src/extension/importer/core/src/main/java/org/geoserver/importer/FeatureDataConverter.java

```diff
@@ -187,6 +187,15 @@ String attName(String name) {
         }
     };
 
+    private static boolean containsAttribute(SimpleFeature ft, String attName) {
+        for (AttributeDescriptor att : ft.getType().getAttributeDescriptors()) {
```

Comment:
Err... if I understand what this does, wouldn't it be the same to check if obj is null or not?